docker/all.Dockerfile

```diff
@@ -4,51 +4,30 @@ LABEL build_date="2020-09-30" \
 	  description="Jolie Documentation" \
 	  url="https://jolie-lang.org"
 MAINTAINER Brian Alberg "alberg@imada.sdu.dk"
-## Fix issue in Node/Docker as reported here:
-## https://stackoverflow.com/questions/52196518/could-not-get-uid-gid-when-building-node-docker
-ENV npm_config_unsafe_perm true
 WORKDIR /home
-RUN mkdir -p /home/nginx/docs
 RUN apk update && \
     apk add git && \
     apk add nodejs npm --no-cache
-RUN npm update
 RUN npm install -g gitbook-cli
-RUN npm install mv
-RUN npm install gitbook-plugin-highlight-jolie
-RUN npm install gitbook-plugin-theme-jolie
-RUN npm install gitbook-plugin-logo
-RUN npm install gitbook-plugin-collapsible-chapters
```

Comment:
By removing these plugins we lose support for jolie syntax highlighting, the theme (which include the version-switcher) and logo. Did you move these somewhere else?

Reply:
`gitbook install` installs from npm all the plugins listed in `book.json`, thus there is no need to manually install them.